src/renderer/viz/expressions/buckets.js

```diff
@@ -4,60 +4,39 @@ import { implicitCast, getOrdinalFromIndex } from './utils';
 /**
  * Given a property create "sub-groups" based on the given breakpoints.
  *
- * Imagine a traffic dataset with a speed property. We want to divide the roads in
- * 3 buckets (slow, medium, fast) based on the speed using a different color each bucket.
+ * This returns a number or category expression depending on the input values.
  *
- * We´ll need:
- *  - A {@link carto.expressions.ramp|ramp} to add a color for every bucket.
- *  - A {@link carto.expressions.palettes|colorPalette} to define de color scheme.
+ * @param {Number|Category} property - The property to be evaluated and interpolated
+ * @param {Number[]|Category[]} breakpoints - Numeric expression containing the different breakpoints.
+ * @return {Number|Category}
  *
- * ```javascript
- *  const s = carto.expressions;
- *  const $speed = s.prop('speed');
- *  const viz = new carto.Viz({
+ * @example <caption>Display a traffic dataset is 3 colors depending on the numeric speed.</caption>
+ * const s = carto.expressions;
+ * const viz = new carto.Viz({
  *    color: s.ramp(
- *      s.buckets($speed, [30, 80, 120]),
+ *      s.buckets(s.prop('speed'), [30, 80, 120]),
  *      s.palettes.PRISM
  *    )
  * });
- * ```
  *
- * ```javascript
- *  const viz = new carto.Viz(`
+ * @example <caption>Display a traffic dataset is 3 colors depending on the numeric speed. (String)</caption>
+ * const viz = new carto.Viz(`
  *    color: ramp(buckets($speed, [30, 80, 120]), PRISM)
  * `);
- * ```
- *
- * Using the buckets `expression` we divide the dataset in 3 buckets according to the speed:
```

Comment:
I think this clarification is useful, although it contains some errors, could you keep it but fixing it?

Reply:
The examples format is one line description and the example. So we can not add a multiline description in the inline example. This extra information could be added in the guides. However, we can extend the description a little bit. What about:

`Display a traffic dataset is 3 colors depending on the numeric speed: (0, 29), (30, 79), (80, 120).`

We could add this in the documentation description:

`Values lower than the minimum will be in the first bucket and values higher than the maximum will be in the last one.`

Reply:
Finally, I have included the info inside the example with comments.

Reply:
Did you push the commits? I don't see it :confused:

Reply:
https://github.com/CartoDB/carto-vl/pull/855/commits/1aad7df8f548d310e8748a2ddfe74cb41cd8ba9a